src/main/java/bio/terra/testrunner/runner/TestRunner.java

```diff
@@ -63,6 +63,13 @@ public TestRunSummary(String id) {
     private String startUserJourneyTimestamp;
     private String endUserJourneyTimestamp;
     private String endTimestamp;
+    // Include the user-provided resourceFileName of test config in the summary:
+    //  perf/**.json, integration/**.json, resiliency/**.json
+    // This can be used to facilitate grouping of test runner results on the dashboard.
+    private String resourceFileName;
```

Comment:
Since this is already in the `TestConfiguration`, I think you don't need it in the summary also, since both should be written out to files.

Reply:
Sure, each test config gets its own directory that gets sent to the GCS as a `tar.gz`. When streaming the `json` files to `BigQuery` with the Cloud Function, only the `SUMMARY_testRun.json` will be streamed because it has the underlying `id` property in it. We are not streaming `json` files without an `id`.